main.go

```diff
@@ -9,26 +10,67 @@ import (
 	"strings"
 )
 
+var version string
+
+// printUsage prints the help message for the program.
+func printUsage() {
+	fmt.Printf("Usage: %s [OPTIONS] <directory>\n", filepath.Base(os.Args[0]))
+	fmt.Println("Options:")
+	fmt.Println("  -h, --help            Show this help message")
+	fmt.Println("  -v, --version         Show the version information")
```

Comment:
how about the simpler `flag.PrintDefaults()` approach?

Reply:
Sounds good